queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name, date_of_birth  FROM animals WHERE name='Agumon' OR name='Pikachu';
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT name, neutered FROM animals WHERE neutered='1';
+
+SELECT * FROM animals WHERE name!='Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT name FROM animals WHERE neutered='1' AND escape_attempts < 3;
```

Comment:
- [ ] Good job by adding some of changes mentioned by the previous reviewer. However as he pointed out, kindly ensure that you also update your screenshots after adding the asked query as right now, there is no one matching the result of the query you added.

Reply:
Hello @tresorsawasawa the screenshot to that query is already added, please check out again the screenshots, I just didn't add the query to the query's file

Reply:
@tresorsawasawa I need you to approve this to move forward, please approve it